general-mechanics/bugs.md

```diff
@@ -25,6 +25,10 @@ Opening the exit menu in the Abyss does not pause the timer. This results in a b
 
 There are inconsistencies in whether a character wielding 4NO can receive the 4-pc Set Bonus on their Elemental Burst or not.
 
+## Domains
+
+Exiting or resetting a domain [will not reset a character's skill cooldown.] (https://github.com/PeonyPuffin/TCL/blob/Resetting-or-Leaving-a-Domain-Does-Not-Reset-Skill-CD/evidence/general-mechanics/bugs.md#domains)
```

Comment:
```suggestion
Exiting or resetting a domain [will not reset a character's skill cooldown.] (../../evidence/general-mechanics/bugs.md#domains)
```
if you link from your fork it wont work correctly when merged, you need to format it like this, each double .. means going back 1 folder from this location, so we go to general mechanics, to the TCL main page, then we can go to evidence.